src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -87,9 +102,40 @@ public static SusiThought  scrapeQuora(String profile) {
 		String rssFeedLink = url + "/rss";
 		quoraProfile.put("rss_feed_link", rssFeedLink);
 
+		Elements userBasicInfo = userHTML.getElementsByAttributeValueContaining("class", "UserCredential IdentityCredential");
+		for (Element info: userBasicInfo) {
+			String infoText = info.text();
+			if (infoText.startsWith("Works")) {
+				quoraProfile.put("works_at", infoText);
```

Comment:
@SKrPl I guess there will be a problem with the method you mentioned. It is not mandatory that all the profiles will have all the three fields that is works_at, studies_at and lives_in. If any of the fields are missing then the approach you suggested will cause problem.For if we try scrape this profile: https://www.quora.com/profile/Susmita-Maitra, then it gives the following output : 
"works_at": "Studied at National Institute of Technology Durgapur",
"studies_at": "Lives in Kolkata, West Bengal, India",